src/plugins/provenance_rename_operator.py

```diff
@@ -86,12 +75,13 @@ def _get_existing_tables(
             )
             table_lookup[to_snake_case(real_tablename)] = table
 
-        snaked_tablenames_str = self._snake_tablenames(table_lookup.keys())
         rows = pg_hook.get_records(
-            f"""
-                SELECT tablename FROM pg_tables
-                WHERE schemaname = '{pg_schema}' AND tablename IN ({snaked_tablenames_str})
             """
+            SELECT tablename FROM pg_tables
+             WHERE schemaname = %(schema)s
+               AND tablename IN %(tables)s
+        """,
+            {"schema": pg_schema, "tables": tuple(table_lookup.keys())},
```

Comment:
Maybe add some extra protection for the situation that `table_lookup` is empty.

Reply:
Good catch as `SELECT ... FROM ... WHERE ... IN ()` is a SQL error. Fixed